src/include/rados/librados.hpp

```diff
@@ -1154,6 +1154,9 @@ namespace librados
 
     config_t cct();
 
+    void set_honor_osdmap_full();
+    void unset_honor_osdmap_full();
+
```

Comment:
In “Objecter::_prepare_osd_op”, “CEPH_OSD_FLAG_FULL_FORCE” is set before sent op to an OSD, if “honor_osdmap_full” is false. When removing an image, several ops, like open, delete, watch, will be sent to the OSD, it’s difficult to set FULL_TRY flag for these ops one by one.